### ISSUE TITLE
JSX: Used import does not generate a dependency

### ISSUE THREAD

**Issue by basarat** [Microsoft/TypeScript#3759]
Consider the following class: 

```tsx
export class MyComponent {  
  render() {}
}
```

If you use it : 

```tsx
import {MyComponent} from "./valuebasedUsage";
var foo = <MyComponent/>
```

emit: 
```js
var foo = React.createElement(valuebasedUsage_1.MyComponent, null);
```

The code generated *does not contain the emit* for `import {MyComponent} from "./valuebasedUsage";` 

/cc @RyanCavanaugh 
Refs : https://github.com/TypeStrong/atom-typescript/issues/442

**Comment by RyanCavanaugh** [comment]
What commit are you on? Should have been fixed in #3675; I can't repro using current bits.

**Comment by basarat** [comment]
> I can't repro using current bits.

Sorry. I should have tried latest :rose: fixed indeed.

**Comment by sergiusignacius** [comment]
Is this fixed in 1.6.2? I have a line that says:

```typescript
import { Dock } from 'react-dock';
```

And the generated code is:
```javascript
var react_dock_1 = require('react-dock');
```
